src/internal/connector/sharepoint/collection_test.go

```diff
@@ -65,9 +65,9 @@ func (suite *SharePointCollectionSuite) TestSharePointDataReader_Valid() {
 	assert.Equal(t, readData, m)
 }
 
-// TestSharePointListCollection tests basic functionality to create
+// TestListCollection tests basic functionality to create
 // SharePoint collection and to use the data stream channel.
-func (suite *SharePointCollectionSuite) TestSharePointListCollection() {
```

Comment:
Could use some tests for page collection here before this gets merged.

Reply:
Resolved with [d4de96f](https://github.com/alcionai/corso/pull/2187/commits/d4de96f6f578442570b18c75d9070cc73761db5e)